test/compress_test.js

```diff
@@ -7,6 +7,15 @@ var fs = require('fs');
 var unzip = require('unzip');
 var tar = require('tar');
 
+function fileExists(filePath) {
```

Comment:
I already fixed all this comments: https://github.com/gruntjs/grunt-contrib-compress/pull/186#pullrequestreview-6430123